File: Backend/src/main/java/com/feedle/feedleapi/Networking/MakeReactionRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.PostReaction;

public class MakeReactionRequest extends Request {
    private PostReaction postReaction;
    public MakeReactionRequest(PostReaction postReaction)
    {
        super(RequestType.MakeReactionRequest);
        this.postReaction = postReaction;
    }

    public PostReaction getPostReaction() {
        return postReaction;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/MakeReactionRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class MakeReactionRequest : Request
    {
        [JsonPropertyName("postReaction")]
        public PostReaction PostReaction { get; set; }
        public MakeReactionRequest(PostReaction postReaction) : base(RequestType.MakeReactionRequest)
        {
            PostReaction = postReaction;
        }
    }
}